src/ExtensionsSample/Program.cs

```diff
@@ -30,13 +30,13 @@ public static void Main(string[] args)
                 filesConfig.RootPath = @"c:\temp\files";
             }
 
-            config.UseFiles(filesConfig);
+            //config.UseFiles(filesConfig);
```

Comment:
That was for testing -- I'll back out.

---
In reply to: [63577537](https://github.com/Azure/azure-webjobs-sdk-extensions/pull/97#discussion_r63577537) [](ancestors = 63577537)